conftest.py

```diff
@@ -214,6 +214,20 @@ def pytest_configure(config):
             ),
         )
 
+    if find_spec("dask") is not None:
+        # this is fixed upstream, it is only necessary here due to a numpy
+        # version conflict in dask and yt. dask version 2.30.0 is the newest
+        # version that will work with numpy 1.14.2 (the minimum for yt).
+        # once the numpy min can be bumped to >= 1.15.1, the min dask version
+        # can be bumped to 2020.12.0 and this filter can be removed
```

Comment:
thank you for documenting this extensively. If it make sense, we could also make the filter conditional, and skip it for sufficiently recent versions of numpy (or dask ?)

Reply:
ya, that's a good idea. I'll try that out.